common/bstrlib/bstrwrap.h

```diff
@@ -75,6 +75,12 @@
 
 #endif
 
+#if defined(_MSC_VER)
```

Comment:
Do not make duplicates here. `bstrwrap.h` includes `bstrlib.h` immediately, there is no need for this macro addition here.

Reply:
Fixed in 582910fb96edefffa7c51442cc39681f90ece010